test/jdk/java/lang/invoke/defineHiddenClass/src/Lambda.java

```diff
@@ -25,7 +25,7 @@
 
 public class Lambda implements HiddenTest {
      public void test() {
-         Function<Object, String> f = Object::toString;
+         Function<Object, String> f = o -> o.toString();
```

Comment:
Recommend you retain the existing method reference e.g.:
```
         Function<Object, String> fmref = Object::toString;
         Function<Object, String> flexp = o -> o.toString();
         String s = fmref.apply(this) + flexp.apply(this);
         ...
```
Or split them out into two tests (easier to debug if case fails).

Reply:
I've split the tests, thanks!